client/src/components/Ticketing/donation/DonorBenefitsPage.tsx

```diff
@@ -0,0 +1,194 @@
+import React, {ReactElement, useState} from 'react';
+import {useNavigate} from 'react-router';
+
+/**
+ * Renders the Donations page without checkout
+ *
+ * @returns {ReactElement}
+ */
+export default function DonorBenefitsPage(): ReactElement {
+  const history = useNavigate();
+  return (
+    <div className='w-full max-w-6xl mx-auto py-[5rem] px-[1rem] tab:px-[5rem] flex flex-col items-center'>
+      {/* Back button*/}
+      <div className='w-full flex flex-row mb-8'>
+        <button
+          onClick={() => history('/donate')}
+          className='bg-blue-500 mt-10 hover:bg-blue-600 px-3 py-2 rounded-xl flex flex-row items-center text-white'
+        >
+          <svg
+            xmlns='http://www.w3.org/2000/svg'
+            className='h-5 w-5'
+            viewBox='0 0 20 20'
+            fill='currentColor'
+          >
+            <path
+              fillRule='evenodd'
+              d='M9.707 14.707a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 1.414L7.414 9H15a1 1 0 110 2H7.414l2.293 2.293a1 1 0 010 1.414z'
+              clipRule='evenodd'
+            />
+          </svg>
+          back to Donation
+        </button>
+      </div>
+      {/* Header text */}
+      <div className='w-full max-w-4xl px-8 py-4 my-5 rounded-sm text-center text-4xl text-zinc-100 bg-indigo-600 font-semibold'>
+        <h1>DONOR BENEFITS</h1>
+      </div>
+      <div className='mb-5 py-4 text-center'>
+        <div>
```

Comment:
Would it be possible to go through and update divs that only contain text to p tags so that they are a little more semantic?

Reply:
This page actually looks like it's trying to reconstruct this [existing page](https://portlandplayhouse.org/donor-benefits/) on Portland Playhouse's website but I can't get it to load in wondertix. PPH can just keep the page that they have on their actual site so we don't need this

Reply:
Removed https://github.com/WonderTix/WonderTix/pull/517/commits/f9635fba6475e863eaf08301b78c89025bd99195